docs/how-to/installation/setup-eks-cluster.txt

```diff
@@ -238,7 +249,7 @@ file such as ``determined-autoscaler.yaml``:
              value: "true"
              effect: NoSchedule
          containers:
-           - image: k8s.gcr.io/autoscaling/cluster-autoscaler:v1.17.3
+           - image: <cluster-autoscaler-image>
```

Comment:
non-blocking: should we reiterate the link to autoscalers here? I have a feeling people might not read...

Reply:
Done!